### ISSUE TITLE
Geologic Unit

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#7]
Proposed definition (from IUGS and OGC GeoSciML) : 
Conceptually, a GeologicUnit may represent a body of material in the Earth whose complete and precise extent is inferred to exist (e.g., North American Data Model GeologicUnit, Stratigraphic unit in the sense of NACSN, or International Stratigraphic Code ), or a classifier used to characterize parts of the Earth (e.g. lithologic map unit like 'granitic rock' or 'alluvial deposit', surficial units like 'till' or 'old alluvium'). It includes both formal units (i.e. formally adopted and named in an official lexicon) and informal units (i.e. named but not promoted to a lexicon) and unnamed units (i.e., recognizable, described and delineable in the field but not otherwise formalised).  In simpler terms, a geologic unit is a package of earth material (generally rock).

**Comment by mbeaufils** [comment]
Additionnal details on the GeologicUnit concept.
The CGI vocabularies introduce (sub-types:)[https://cgi.vocabs.ga.gov.au/object?uri=http%3A//resource.geosciml.org/classifier/cgi/geologicunittype/geologic_unit].

**Comment by dponti** [comment]
I'm satisfied with the definition here. I think the question for us to resolve is with the definition of the geotechnical unit with respect to a geologic unit - does it stand alone or is it best defined as a specialization of geologic unit based on properties of interest, much like a geophysical unit is considered by CGI as a type of geological unit.

**Comment by neilchadwick-dg** [comment]
@jjkaelin I sympathise with your concern on terminology. Unit vs 'stratum' is something I researched for AGSi. I found that *unit* was probably the correct term over stratum, but I see that many people (mainly engineers) always use stratum. 'Formation' I thought was reserved as part of the 'official' geological classification system for a country, which may not always correspond one to one with the 'unit' that the project team happens to be using (it is acceptable for them to differ). But I should probably defer to the geologists on this!

Generally, I'm happy with geologicunit. My recollection is that geologicunit in the CGI vocab was a high level catch-all term. In practice, for most uses cases that we are dealing with geologicunit will be sufficient.  If very detailed geological work is done then some of the other unit types may come into play, but I see those as a minority use cases (more relevant to GeoSciML).

My (and the AGSi) view is that a geologicunit is part of an observational model. It is usually the 'best guess' of what is going on in the ground. The classification will normally be driven by the engineering geology. Uncertainty is inherent, of course.

In the majority of cases, the geologicunit will not be directly used in design.

It is the *geotechnicalunit* that will be used in analysis/design. This is normally derived from the geologic units. The classification system may be the same, or it may differ to suit to ground and/or analysis method. The extents are likely to differ to take account of uncertainty, i.e. assessing 'characteristic' extends. Parameters for design can be assigned.

A geological model with geological units may give birth to several different geotechnical models with different geotechnical  units (or at least different parameters).



.

**Comment by jjkaelin** [comment]
neilchadwick-dg wrote:
> In the majority of cases, the geologicunit will not be directly used in design 

This is what is bothering me. In my view it seems obvious that all our terminology should be usable in design. It may not be the local "dialect" or the project "lingo". However it should be terminology that one can use in a design report without hesitation.

Am I missing something here? I believer that I am very familiar with the differences between geological 'units' and geotechnical 'units', so that is not my issue here.

**Comment by PieGARNIER** [comment]
Personnaly, I like the "Unit" term. I think it reflects at some point a geological or geotechnical interpretation, where "stratum" of "formation" may be understood as "closed" objects (commonly associated to a horizontal layer of homogenous soil properties, but what soil & what proprerties?). "Unit" also gives flexibility, since it can refer to various "scopes" : for example a same soil volume may be an overlay of various geotechnical unit, depending on what focus is given (mechanical, granulometric, etc.). 

The way I see it, since each "unit" will refer to a specific model (geotechnical, geological, design model), I think it will be understandable.

**Comment by mbeaufils** [comment]
Having a deeper look to the [type of GeologicUnit](https://inspire.ec.europa.eu/codelist/GeologicUnitTypeValue/geologicUnit) as defined by CGI and reused in the INSPIRE Directive.

Also considering that the GeologicalUnit in a generic term, do we target a specific GeologicUnit for the geological model? eg. LithostratigraphicUnit?

**Comment by Didymograptus** [comment]
I suggest that at the moment we don't target a specific type. Let's define what we /the industry needs first and then see if it fits with an established definition. There is no need to constrain our thoughts at the moment.
